src/Spectre.Algorithms/Results/DivikResult.cs

```diff
@@ -93,9 +98,19 @@ internal DivikResult(object matlabResult)
             Merged = ParseMerged(array);
             Subregions = ParseSubregions(array);
         }
+
+        /// <summary>
+        /// Empty constructor for serialization purposes.
+        /// </summary>
+        public DivikResult()
```

Comment:
Unlike I thought (I blame it on java) it can actually be even _private_. I'll change that.